### DISCUSSION TITLE
How do you run/deploy Storybook 7 on a subpath?

### DISCUSSION THREAD

**Discussion by AlanMorel** [storybookjs/storybook#22280] [Help]
In my `main.ts`, I'm exporting a StorybookConfig, however, there doesn't seem to be any straightforward way to run and deploy Storybook 7 on a subpath.

Here's some example code if it helps:

```typescript
import type { StorybookConfig } from "@storybook/nextjs";

const path = "/example/storybook/";

const config: StorybookConfig = {
    stories: ["../src/**/*.stories.@(ts|tsx)"],
    addons: [
        "@storybook/addon-links",
        "@storybook/addon-essentials",
        "@storybook/addon-interactions",
    ],
    framework: {
        name: "@storybook/nextjs",
        options: {}
    },
    docs: {
        autodocs: "tag"
    },
};

export default config;
```

**Comment by arodik** [comment]
I have the same problem. Tried different approaches described here https://github.com/storybookjs/storybook/issues/1291, but they make the situation even worse

Would be great to add some field into configuration for base path

**Reply by bluessista** [reply]
Same problem appears here - we also would like to be able to set it locally to enable devs and designers running the app and the docs (SB) in parallel in our mono repo.
